docs/formats/sly_pointcloud_user_manual.md

```diff
@@ -0,0 +1,183 @@
+# Supervisely Point Cloud user manual
+
+## Contents
+
+- [Format specification](#format-specification)
+- [Import Supervisely Point Cloud](#import-supervisely-point-cloud-dataset)
+- [Export to other formats](#export-to-other-formats)
+- [Export to Supervisely Point Cloud](#export-to-supervisely-point-cloud)
+- [Examples](#examples)
+
+## Format specification
+
+Point Cloud data format:
+- [specification](https://docs.supervise.ly/data-organization/00_ann_format_navi).
+- [example](https://drive.google.com/file/d/1BtZyffWtWNR-mk_PHNPMnGgSlAkkQpBl/view).
+
+This dataset format supports the following types of annotations:
+- `cuboid_3d`
+
+Supported annotation attributes:
+- `object` (read/write, integer),
+- `createdAt` (write, string),
+- `updatedAt` (write, string),
+- `labelerLogin` (write, string), responsible for the corresponding fields
+  in the annotation file.
+- arbitrary attributes
+
+Supported image attributes:
+- `description` (read/write, string),
+- `createdAt` (write, string),
+- `updatedAt` (write, string),
+- `labelerLogin` (write, string), responsible for the corresponding fields
+  in the annotation file.
+- `frame` (read/write, integer). Indicates frame number of the image.
+- arbitrary attributes
+
+## Import Supervisely Point Cloud dataset
+
+An example datast in Supervisely Point Cloud format is available for download:
+
+<https://drive.google.com/u/0/uc?id=1BtZyffWtWNR-mk_PHNPMnGgSlAkkQpBl&export=download>
+
+Point Cloud dataset directory should have the following structure:
+
+<!--lint disable fenced-code-flag-->
+```
+└─ Dataset/
+    ├── ds0/
+    │   ├── ann/
+    │   │   ├── <pcdname1.pcd.json>
+    │   │   ├── <pcdname2.pcd.json>
+    │   │   └── ...
+    │   ├── pointcloud/
+    │   │   ├── <pcdname1.pcd>
+    │   │   ├── <pcdname1.pcd>
+    │   │   └── ...
+    │   ├── related_images/
+    │   │   ├── <pcdname1_pcd>/
+    │   │   |  ├── <image_name.ext.json>
+    │   │   |  ├── <image_name.ext.json>
+    │   │   └── ...
+    ├── key_id_map.json
+    └── meta.json
+```
+
+There are two ways to import Supervisely Point Cloud dataset:
+
+```bash
+datum import --format sly_pointcloud --input-path <path/to/dataset>
+# or
+datum create
+datum add path -f sly_pointcloud <path/to/dataset>
+```
+
+To make sure that the selected dataset has been added to the project,
+you can run `datum info`, which will display the project and dataset
+information.
+
+## Export to other formats
+
+Datumaro can convert Supervisely Point Cloud dataset into any other
+format [Datumaro supports](../user_manual.md#supported-formats).
+
+Such conversion will only be successful if the output
+format can represent the type of dataset you want to convert,
+e.g. 3d point clouds annotations can be
+saved in `KITTI Raw` format, but not as `COCO keypoints`.
```

Comment:
The inline code style seems inappropriate here. These aren't identifiers or code fragments.